buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -269,34 +270,36 @@ class PerformanceTestPlugin : Plugin<Project> {
             createDistributedPerformanceTestTask(name, clazz, performanceSourceSet, prepareSamplesTask).configure(configure)
         }
 
+        val channelSuffix = if (OperatingSystem.current().isLinux) "" else "-${OperatingSystem.current().name.toLowerCase()}"
+
         create("distributedPerformanceTest", DistributedPerformanceTest::class) {
             includeCategories(performanceRegressionTestCategory)
             excludeCategories(slowPerformanceRegressionTestCategory)
-            channel = "commits"
+            channel = "commits$channelSuffix"
             retryFailedScenarios()
         }
         create("distributedSlowPerformanceTest", DistributedPerformanceTest::class) {
             includeCategories(slowPerformanceRegressionTestCategory)
-            channel = "commits"
+            channel = "commits$channelSuffix"
```

Comment:
Instead of using a postfix for the channel, I think it would be much better to use a separate column in the performance database (e.g. `os`) similar to `branch`. Can you make that change?